packages/typespec-ts/test/modularIntegration/generated/client/structure/default/src/api/operations.ts

```diff
@@ -26,13 +33,19 @@ import {
 export function _oneSend(
   context: Client,
   options: OneOptions = { requestOptions: {} }
-): StreamableMethod<One204Response> {
+): StreamableMethod<One204Response | OneDefaultResponse> {
   return context
     .path("/one")
     .post({ ...operationOptionsToRequestParameters(options) });
 }
 
-export async function _oneDeserialize(_result: One204Response): Promise<void> {
```

Comment:
I remeber there's a logic handing the rename from result to _result, can you remove this logic in the codegen if default response always exists.